phpunit.xml.dist

```diff
@@ -14,6 +14,7 @@
     <testsuites>
         <testsuite name="AuditStash Test Suite">
             <directory>./tests/TestCase</directory>
+            <exclude>./tests/TestCase/Persister/ElasticSearchPersisterTest.php</exclude>
```

Comment:
This test fails on local. I suggest creating a separate phpunit.xml without this line that runs in a github action pipeline. It looks like you had travis setup, but I don't know if its working still? I am fine helping out on this even though I don't use elastic.